pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -228,31 +228,32 @@ func (pl *DefaultPreemption) SelectVictimsOnNode(
 // considered for preemption.
 // We look at the node that is nominated for this pod and as long as there are
 // terminating pods on the node, we don't consider this for preempting more pods.
-func (pl *DefaultPreemption) PodEligibleToPreemptOthers(pod *v1.Pod, nominatedNodeStatus *framework.Status) bool {
+func (pl *DefaultPreemption) PodEligibleToPreemptOthers(pod *v1.Pod, nominatedNodeStatus *framework.Status) (bool, string) {
 	if pod.Spec.PreemptionPolicy != nil && *pod.Spec.PreemptionPolicy == v1.PreemptNever {
-		klog.V(5).InfoS("Pod is not eligible for preemption because it has a preemptionPolicy of Never", "pod", klog.KObj(pod))
```

Comment:
`klog.KObj(pod)`  logs as {podNamespace}/{podName}, should keep this here?

Reply:
sg.  I add the klog.KObj(pod) here
```go
	ok, msg := ev.PodEligibleToPreemptOthers(pod, m[pod.Status.NominatedNodeName])
	if !ok {
		klog.V(5).InfoS(msg, "pod", klog.KObj(pod))
		return nil, framework.NewStatus(framework.Unschedulable, msg)
	}
```

Reply:
SG